pkg/github/client.go

```diff
@@ -2078,10 +2079,11 @@ func (c *client) GetFailedActionRunsByHeadBranch(org, repo, branchName, headSHA
 	// See https://docs.github.com/en/rest/actions/workflow-runs?apiVersion=2022-11-28#list-workflow-runs-for-a-workflow
 	// This makes it hard to use directly. Instead, we loop through the runs and check them individually.
 	// A successful workflow will have status "completed" and conclusion "success".
+	// A skipped workflow will have status "completed" and conclusion "skipped".
 	// A failed workflow also have status "completed", but the conclusion can be either "failure" or "cancelled".
-	// We only want completed jobs that are not successful.
+	// We only want completed jobs that are not skipped and not successful.
 	for _, run := range runs.WorkflowRuns {
-		if run.Status == "completed" && run.Conclusion != "success" {
+		if run.Status == "completed" && !slices.Contains([]string{"success", "skipped"}, run.Conclusion) {
```

Comment:
With only 2 values to check this seems like overkill, but if you really want to use `slices.Contains` you should make a const slice for the values to check outside of the function.

Personally, I would just use check both values individually.

Reply:
GitHub Actions workflows return a conclusion value after completion. The possible conclusion values are:
- `success`: The workflow completed successfully
- `failure`: The workflow failed due to test failures, code errors, etc.
- `cancelled`: The workflow was manually cancelled
- `skipped`: The workflow was skipped (not run)
- `timed_out`: The workflow exceeded the maximum allowed runtime
- `action_required`: Requires a user action before continuing
- `neutral`: The workflow completed but neither succeeded nor failed (neutral outcome)

So I think the passed values are `["success", "skipped"]`, the `action_required` should be treat as failed state in triggering.